notifier.rst

```diff
@@ -365,6 +365,7 @@ Service
                                          **DSN**: ``sns://ACCESS_KEY:SECRET_KEY@default?region=REGION``
 `Bluesky`_                               **Install**: ``composer require symfony/bluesky-notifier`` \
                                          **DSN**: ``bluesky://USERNAME:PASSWORD@default``
+                                         **Extra properties in SentMessage**:: ``cid``
```

Comment:
Not sure its worth do documents such internal things

Reply:
Yes, we can discuss about this. I added it here for consistency because we added the extra SentMessage info of other integrations before. Thanks.